go/enclave/l2chain/l2_chain.go

```diff
@@ -472,8 +472,8 @@ func (oc *ObscuroChain) produceGenesisBatch(blkHash common.L1BlockHash) (*core.B
 func (oc *ObscuroChain) processState(batch *core.Batch, txs []*common.L2Tx, stateDB *state.StateDB) (common.L2BatchHash, []*common.L2Tx, []*types.Receipt, []*types.Receipt) {
 	var executedTransactions []*common.L2Tx
 	var txReceipts []*types.Receipt
-
-	txResults := evm.ExecuteTransactions(txs, stateDB, batch.Header, oc.storage, oc.chainConfig, 0, oc.logger)
+	limiter := core.NewBatchSizeLimiter(25_000, *oc.crossChainProcessors.Local.GetBusAddress(), crosschain.CrossChainEventID)
```

Comment:
can we put this 25_000 into a constant just so the limit is easy to read/adjust? (Maybe with a comment explaining why that number)